firefox-ios/Client/Frontend/Browser/StartAtHomeHelper.swift

```diff
@@ -45,24 +45,20 @@ class StartAtHomeHelper: FeatureFlaggable {
     /// should perform its function.
     public func shouldStartAtHome() -> Bool {
         let setting = startAtHomeSetting
-        guard setting != .disabled else { return false }
 
         let lastActiveTimestamp = UserDefaults.standard.object(forKey: "LastActiveTimestamp") as? Date ?? Date()
         let dateComponents = Calendar.current.dateComponents([.hour, .second],
                                                              from: lastActiveTimestamp,
                                                              to: Date())
-        var timeSinceLastActivity = 0
-        var timeToOpenNewHome = 0
 
-        if setting == .afterFourHours {
-            timeSinceLastActivity = dateComponents.hour ?? 0
-            timeToOpenNewHome = 4
-        } else if setting == .always {
-            timeSinceLastActivity = dateComponents.second ?? 0
-            timeToOpenNewHome = 5
+        switch setting {
```

Comment:
LGTM just a nit about code structure. What do you think if we refactor it like this:
```swift
switch startAtHomeSetting {
case .afterFourHours:
  let lastActivityTimestamp = ...
  let dateComponents = ..
  return dateComponents.hour >= 4
case .always:
  return true
case .disabled:
  return false
}
```
I thinks this is more compact but still preserve readability. Let me know what you think